### PR TITLE
Chip to pass

### PR DESCRIPTION
**Pull request by Alan-Penkar**
Fixed chipping in sim.
Changed Coordinated Pass to use a chip by default.

### CODE REVIEW

File: soccer/gameplay/skills/pivot_kick.py [hunk 1]
```diff
@@ -215,6 +215,7 @@ def execute_kicking(self):
         self.set_aim_params()
         vel = (self.aim_target_point - self.robot.pos).normalized(0.5)
         self.robot.set_world_vel(vel)
+        print(self.robot.has_chipper())
```
Comment (tjones320) [inline soccer/gameplay/skills/pivot_kick.py]: remove this

File: soccer/Robot.cpp [hunk 2]
```diff
@@ -347,6 +347,7 @@ void OurRobot::_kick(uint8_t strength) {
 }
 
 void OurRobot::_chip(uint8_t strength) {
+    std::cout << "YAYYAYAYAYA" << std::endl;
```
Comment (jgkamat) [inline soccer/Robot.cpp]: plzz

File: soccer/radio/SimRadio.cpp [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
 #include <protobuf/grSim_Commands.pb.h>
 #include <protobuf/grSim_Packet.pb.h>
 #include <Geometry2d/Util.hpp>
 #include <Network.hpp>
 #include <Robot.hpp>
 #include <Utils.hpp>
 #include <stdexcept>
+#include <math.h>
+
+#define PI 3.14159265
```
Comment (jgkamat) [inline soccer/radio/SimRadio.cpp]: For future ref, you can use `M_PI` in geometry2d

File: soccer/Robot.cpp (hunk 2 above)
Comment (Alan-Penkar) [reply soccer/Robot.cpp]: we're just really excited about our new chipping capability..

**Pull request merged**